js/ui/components/clubhouse.js

```diff
@@ -653,7 +653,8 @@ function _migrateHack1() {
 
     // Only enable the first time, to allow the user to disable the hack mode
     try {
-        Gio.File.new_for_path('.hack1-migrated').create(Gio.FileCreateFlags.NONE, null);
+        const filePath = `${GLib.get_user_config_dir()}/.hack1-migrated`;
```

Comment:
let path = GLib.build_filenamev([GLib.get_user_config_dir(), '.hack1-migrated']);

Would be an option too, looking at other places in the shell.

Reply:
Yes, I think that's a better option, let's use that instead.